auto_submit/analysis_options.yaml

```diff
@@ -9,4 +9,5 @@ linter:
     constant_identifier_names: false # we have all capitalized enums in check_for_waiting_pull_requests_test.dart
     require_trailing_commas: true
     unawaited_futures: true
-    prefer_final_fields: true
\ No newline at end of file
+    prefer_final_fields: true
+    prefer_final_locals: true
```

Comment:
Is this something we can add in https://cs.opensource.google/flutter/cocoon/+/main:analysis_options.yaml?

Generally, we want all the projects in Cocoon to follow the same style guide

Reply:
Yeah I can add this there. No problem.